jenkins-slaves/jenkins-slave-ansible/Dockerfile

```diff
@@ -1,14 +1,28 @@
-FROM openshift/jenkins-slave-base-centos7:latest
+FROM openshift/jenkins-slave-base-centos7
 
-ENV ANSIBLE_VERSION=2.5.8 \
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable" \
+	ANSIBLE_VERSION=2.7.0 \
     ANSIBLE_REVISION=1
 
-RUN INSTALL_PKGS="python2-pip" && \
+RUN INSTALL_PKGS="java-1.8.0-openjdk-devel.x86_64 git tree vim unzip curl python2-pip" && \
```

Comment:
Please add `scl-utils` to `INSTALL_PKGS` or else you'll see the following message
```
/usr/local/bin/scl_enable: line 3: scl_source: No such file or directory
```